London | May-2025| Hakan_Murat_Kavut | Form_Control

hmkavut opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

Form items (name, email, colour and size) added 
Footer is updated

## Questions

Ask any questions you have for your reviewer.

cjyuan commented:
A well-prepared PR makes it easier for reviewers to approve it with minimal back-and-forth.
The "PR Essentials" file on the [Canvas](https://codeyourfutur-yov6609.slack.com/archives/C07QX99JK7B) in the #cyf-code-review channel has some helpful tips to make your PRs more robust and ready for review.
Can you take a few minutes to check against the essentials, including how to prepare PR description? Doing so can help speed up the review process.

And as a practice to get better at using AI tools, try running your code by ChatGPT. You can ask ChatGPT
```
review HTML code:
...        <--- paste your HTML code here
```

Then take a look at the suggestions it gives you, and pick out the ones that actually make sense for your project. It's a great way to learn and improve your work.

Once you've finished making changes, please leave a comment on this PR so I’ll get notified. Thanks!

hmkavut commented:
Hi @cjyuan,

I updated Index.html and fixed errors. I asked to ChatGPT. Then, I modified the file as recommended.

Best,
![Form Control Lighthouse Score](https://github.com/user-attachments/assets/1bc94d69-83cf-4d85-bfa9-5685b4a4d050)

cjyuan reviewed (commented):
Code is error free and well indented. Good job!

I left some requests and suggestions in the code. 

---
Note:
A deleted file will still be recognized as a modified file.
![image](https://github.com/user-attachments/assets/efd66573-4478-4a6b-9a16-216ae823a976)

One possible way to make `Wireframe/index.html` not showing up as a modified file is, instead of deleting the file, replace it by the original copy of the file in the `main` branch.

hmkavut commented on Form-Controls/index.html:
```diff
@@ -1,27 +1,59 @@
+      </div>
+      <div>
+        <br>
+        <button type="reset">Reset</button>
+        <button type="submit">Submit</button>
```
I asked to ChatGPT and there are 3 reason which are User expectations, Accidental clicks and Accessibility. I changed the order.

cjyuan replied:
ChatGPT also ask me to considered whether to include a Reset button or not.

How often do you see a Reset button on a form? When did you last use a Reset button on a form? :)

hmkavut commented on Form-Controls/index.html:
```diff
@@ -1,27 +1,59 @@
+      <br>
+      <div>
+        <label for="customer-name">Customer Name</label>
+        <input type="text" name="customer-name" id="customer-name" required>
```
I updated customer name so white space is not accepted.

hmkavut commented:
Hi @cjyuan 

PR is updated based on your comments.

Best

cjyuan commented:
Changes look good! 

New code does not introduce any error. :D
